### PR TITLE
feat: cache docker images

### PR DESCRIPTION
**Pull request by denis-ev**

### CODE REVIEW

File: group_vars/all [hunk 1]
```diff
@@ -255,8 +255,9 @@ smtp:
 traefik:
   additional_env_vars:
 bastion:
-  wireguard:
```
Comment (codefriar) [inline group_vars/all:258]: Why take this out?

Comment (codefriar) [reply group_vars/all:258]: Nvme mind

File: roles/vivumlab_config/templates/config.yml [hunk 2]
```diff
@@ -67,14 +67,14 @@ bastion:
     port: {{ bastion.wireguard.port | default("51820") }}
     keepalive: {{ bastion.wireguard.keepalive | default("25") }}
   ports_to_forward:
-    2222: 22
```
Comment (codefriar) [inline roles/vivumlab_config/templates/config.yml:70]: These ports should remain. They're needed for smtp and ssh

**Review by codefriar — CHANGES_REQUESTED** [review]

File: roles/vivumlab_config/templates/config.yml (hunk 2 above)
Comment (denis-ev) [reply roles/vivumlab_config/templates/config.yml:70]: I made the port 2222 a custom port, which would be the same as the variable vlab_port.
And I removed the others for security, they would be forwarded from a public accessible server to your own and maybe when the certain services are not running, not even used.
If you want any port, we can add them with an if so they would only be there if needed. 
And with the for loop underneath, everything you have already in there will stay there, if you added stuff before, that would just be gone, now it's saved and not overwritten.

Comment (codefriar) [reply roles/vivumlab_config/templates/config.yml:70]: Gotcha. Will test tonight.

**Comment by codefriar** [comment]
@denis-ev  - give these changes a test please.

**Comment by codefriar** [comment]
@denis-ev  - I feel some urgency on this one, as right now vlab update will nuke any existing bastion server in a way that is not easily recoverable without an out of band console access to the machine - it literally will cause a nat/snat/dnat redirection of port 22 to port 22 which... effectively blocks all incoming port 22 traffic

**Comment by denis-ev** [comment]
@codefriar this one should be good to go, if you could test it again?

**Review by codefriar — APPROVED** [review]

**Pull request merged**